backend/validators/social_media_validator.py

```
@@ -438,8 +445,11 @@ async def _validate_social_media_health(self, validation_result: Dict) -> Dict:
                 "content_generation_working": False
             }
             
-            db = await get_db()
-            conn = await db.get_connection()
+            if not db_manager:
+                logger.error("Database manager not available")
+                return health_status
```

Ruff (0.12.2)

Undefined name `health_status`

(F821)